st3/sublime_lib/resource_path.py

```diff
@@ -171,13 +201,12 @@ def root(self):
     @property
     def package(self):
         """
-        The name of the package the path is within,
-        or ``None`` if the path is not beneath ``'Packages'``.
+        The name of the package the path is within.
         """
-        if self.root == 'Packages' and len(self._parts) >= 2:
+        if len(self._parts) >= 2:
             return self._parts[1]
         else:
-            return None
+            return ''
```

Comment:
What's the rationale for this change?

If we're about semantics, a `None` more clearly indicates that the path didn't contain a package rather than an empty string. In reality, you'd most likely just `if path.package:` anyway, though.

Reply:
Agreed

Reply:
Please also add this to the docstring.

Reply:
Fixed.